fix_migrations.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
is this worth committing? I can remove it. It's from @cincodenada 's proposed fix for my migrations problem.

Reply:
this is great! I think it would make the most sense to have this available separately with the original issue (in a gist or in a comment body) rather than in this PR, since it shouldn't effect people who clone the repo any time after the merge migration was created.